src/pages/index.tsx

```diff
@@ -1,41 +1,371 @@
+import '@uppy/core/dist/style.min.css';
+import '@uppy/dashboard/dist/style.min.css';
 
-function HomepageHeader() {
-	const { siteConfig } = useDocusaurusContext();
-	return (
-		<header className={clsx('hero hero--primary', styles.heroBanner)}>
-			<div className="container">
-				<h1 className="hero__title">{siteConfig.title}</h1>
-				<p className="hero__subtitle">{siteConfig.tagline}</p>
-				<div className={styles.buttons}>
-					<Link
-						className="button button--secondary button--lg"
-						to="/docs/intro"
-					>
-						Docusaurus Tutorial - 5min ⏱️
-					</Link>
-				</div>
-			</div>
-		</header>
-	);
+const dashboardCode = `
+import Uppy  from '@uppy/core'
+import Dashboard from '@uppy/dashboard'
+import RemoteSources from '@uppy/google-drive'
+import ImageEditor from '@uppy/image-editor'
+import Webcam from '@uppy/webcam'
+import Tus from '@uppy/tus'
+
+const uppy = new Uppy()
+  .use(Dashboard, { target: '.DashboardContainer', inline: true })
+  .use(RemoteSources, { companionUrl: "http://companion.uppy.io" })
+  .use(Webcam, { target: Dashboard })
+  .use(ImageEditor, { target: Dashboard })
+  .use(Tus, { endpoint: 'https://tusd.tusdemo.net/files/' })
+`;
+
+const reactCode = `
+import React, { useEffect } from 'react'
+import Uppy from '@uppy/core'
+import Webcam from '@uppy/webcam'
+import { Dashboard } from '@uppy/react'
+
+const uppy = new Uppy().use(Webcam)
+
+function Component () {
+  return <Dashboard uppy={uppy} plugins={['Webcam']} />
+} 
+`;
+
+const vueCode = `
+<template>
+  <div id="app">
+    <dashboard :uppy="uppy" :plugins="['Webcam']" :props="{theme: 'light'}" />
+  </div>
+</template>
+
+<script>
+import { Dashboard } from '@uppy/vue'
+
+import '@uppy/core/dist/style.css'
+import '@uppy/dashboard/dist/style.css'
+
+import Uppy from '@uppy/core'
+import Webcam from '@uppy/webcam'
+
+export default {
+  name: 'App',
+  components: {
+    Dashboard
+  },
+  computed: {
+    uppy: () => new Uppy().use(Webcam)
+  },
+  beforeDestroy () {
+    this.uppy.close({ reason: 'unmount' })
+  }
 }
+</script>
+`;
+
+const angularCode = `
+import { NgModule } from '@angular/core'
+import { UppyAngularDashboardModule } from '@uppy/angular'
+
+import { BrowserModule } from '@angular/platform-browser'
+import { AppComponent } from './app.component'
+
+@NgModule({
+  declarations: [
+    AppComponent,
+  ],
+  imports: [
+    BrowserModule,
+    UppyAngularDashboardModule,
+  ],
+  providers: [],
+  bootstrap: [AppComponent],
+})
+class {}
+`;
+
+const svelteCode = `
+<main> 
+  <Dashboard 
+      uppy={uppy} 
+      plugins={["Webcam"]}
+  />
+</main>
+
+<script>
+import { Dashboard } from '@uppy/svelte'
+
+import Uppy from '@uppy/core'
+import Webcam from '@uppy/webcam'
+
+const uppy = new Uppy().use(Webcam);
+</script>
+`;
+
+const providersIcons = [
+	'box.svg',
+	'unsplash.svg',
+	'googledrive.svg',
+	'dropbox.svg',
+	'instagram.svg',
+	'facebook.svg',
+	'onedrive.svg',
+];
```

Comment:
Let's try the new colorful icons here? Believe it's going to look better, and we won't fail with following some brand guidelines.

Reply:
I still don't think it will look better and brand compliance is important for applications that need to go through approval. In our case that's the Uppy Dashboard, not our homepage.

Reply:
Could we just try and compare tho?

Reply:
Resolved in Slack: keeping it as is.